api/v1/swagger.yaml

```diff
@@ -6160,6 +6160,239 @@ paths:
           $ref: "#/responses/genericError"
 
 # =============================================================================
+  /bots:
+    x-swagger-router-controller: bots
+    get:
+      summary: Retrieve all bots
```

Comment:
"summary" attribute is ok as sentence fragment but "description" attribute should be written as full sentence (or paragraph).